esp/public/media/scripts/custom_form.js

```diff
@@ -865,9 +865,10 @@ var renderNormalField=function(item, field_options, data){
 			type:"text",
 			size:"30"
 		});
+        $new_elem = $new_elem.add($j("<span class='correct_answer'> Correct answer: " + $j('#textField_correct_answer').val() + "</span>"));
```

Comment:
For the text questions, if the admin doesn't specify a correct answer, can we hide the "Correct answer:" in the preview? It's just blank and confusing if there was no correct answer specified